bin/hipcc

```diff
@@ -116,20 +117,26 @@ if ($HIP_PLATFORM eq "hcc") {
 
         # Only include the libstdc++ headers and libraries flags explicitly if the g++ is older than version 5.
         # That's because HCC already uses libstdc++ by default if a newer g++/libstdc++ is available
-        if (${GCC_CUR_VER} eq ${GPP_CUR_VER} and $GPP_VER_FIELDS[0] < 5) {
+        # Cent OS 7 cannot use libstdc++ for compilation, defaults to libc++
+        if (${GCC_CUR_VER} eq ${GPP_CUR_VER} and $GPP_VER_FIELDS[0] < 5 and ($HOST_OSNAME ne "\"centos\"")) {
```

Comment:
we would need to apply the same logic for rhel 7.4

Reply:
What is the name of the OS for rhel 7.4? Is it "RHEL"? I cannot find an official docker image of it.

Reply:
I will try to test on a random docker image of rhel 7.X

Reply:
Oh I figured it out. Seems to be using docker pull registry.access.redhat.com/rhel7. From https://access.redhat.com/containers/?count=10&tab=images&platform=docker#/registry.access.redhat.com/rhel7